Source/Krypton Components/Krypton.Toolkit/Controls Visuals/VisualPopup.cs

```diff
@@ -512,7 +512,10 @@ protected override void OnMouseMove(MouseEventArgs e)
             if (!IsDisposed)
             {
                 // Do we have a manager for processing mouse messages?
-                ViewManager?.MouseMove(e, new Point(e.X, e.Y));
+                if (ViewManager != null)
```

Comment:
Why has this code been expanded into this, the other way is the same and more succinct.

Reply:
Reverted